Userland/Libraries/LibWeb/CSS/Parser/DeprecatedCSSParser.cpp

```diff
@@ -214,7 +214,19 @@ static CSS::Length parse_length(const CSS::ParsingContext& context, const String
 
 static bool takes_integer_value(CSS::PropertyID property_id)
 {
-    return property_id == CSS::PropertyID::ZIndex || property_id == CSS::PropertyID::FontWeight;
+    return property_id == CSS::PropertyID::ZIndex || property_id == CSS::PropertyID::FontWeight || property_id == CSS::PropertyID::Custom;
+}
+
+static StringView parse_custom_property_name(StringView value)
```

Comment:
No need to copy this either.

```suggestion
static StringView parse_custom_property_name(const StringView& value)
```

Reply:
Turns out this is harmless